### PR TITLE
Drop Mysql support

### LINKED ISSUE #234: Drop Mysql support
**Describe the bug**
Supporting two databases causes unnecessary hurdles in development. We decided to drop Mysql support and just use PostgreSQL in all environments

**Are you working on this issue? (Yes/No)**
No

### PR DESCRIPTION
**Pull request by Amansingla97**
Fixes #234 
kindly review it and let me know if something needs to be included :)

### CODE REVIEW

File: app/views/users/sessions/new.html.erb [hunk 1]
```diff
@@ -1,5 +1,6 @@
```
Comment (ankitkataria) [inline app/views/users/sessions/new.html.erb]: Is this change intentional?

File: config/database.example.yml [hunk 2]
```diff
@@ -5,16 +5,17 @@
...
-  adapter: mysql2
-  encoding: utf8
+  adapter: postgresql
+  encoding: unicode
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  username: root
-  password: root
-  socket: /tmp/mysql.sock
+  username: postgres
+  password: postgres
```
Comment (ankitkataria) [inline config/database.example.yml]: Most of the times the password for the postgres user will be empty, Maybe we can replace this with `''`

File: README.md [hunk 3]
```diff
@@ -28,8 +28,8 @@ Please go through the [Contribution Guidelines](CONTRIBUTING.md) before going fo
 **Note:** You might want to use the docker instructions if you do not want to setup your own environment.
 
 * Install ruby using RVM, use ruby-2.5.1
-* Install Dependencies: `bundle install --with mysql` (If you wanto use pg use `bundle install --with pg`)
-* Configure your DB in config/database.yml, copy config/database.yml.example
+* Install Dependencies: `bundle install --with pg`
```
Comment (ankitkataria) [inline README.md]: I guess `--with pg` will not be required now as it will the be the default db for all the environments(test, prod and dev).

**Comment by ankitkataria** [comment]
If mysql is not longer supported I think `database.mysql.example.yml` can be safely removed.

**Comment by Amansingla97** [comment]
ya --pg is not necessaery and i renamed the database.mysql.example.yml to database.example.yml

File: config/database.example.yml [hunk 4]
```diff
@@ -5,16 +5,17 @@
...
+
 
 development:
   <<: *default
   database: circuitverse_development
+  host: localhost
```
Comment (ankitkataria) [inline config/database.example.yml]: test environment will also have `host: localhost`, maybe this can be included in `default` and mentioned in both development and test sections. Production can have its own section.

File: config/database.example.yml (hunk 2 above)
Comment (Amansingla97) [reply config/database.example.yml]: I think the default is postgres . we can add a note in readme to ask the user to check with ''

Comment (ankitkataria) [reply config/database.example.yml]: It says here that [here](https://www.liquidweb.com/kb/what-is-the-default-password-for-postgresql/) isn't a default password initially. Maybe mentioning it as `password: <password>` along with a note in the readme about filling in the field can work. :slightly_smiling_face:

Comment (Amansingla97) [reply config/database.example.yml]: I think it differs with operating systems :) Not sure though
password: <password> along with a note  is the best option

**Comment by danllin** [comment]
Amansingla97:sql_to_posT

**Comment by danllin** [comment]
Amansingla97:sql_to_post

**Comment by danllin** [comment]
Amansingla97:sql_to_post

File: config/database.example.yml [hunk 5]
```diff
@@ -5,17 +5,19 @@
...
-  adapter: mysql2
-  encoding: utf8
+  adapter: postgresql
+  encoding: unicode
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  username: root
-  password: root
-  socket: /tmp/mysql.sock
+  username: postgres
+  password: <password>
```
[the remark this answers is not in the document]
Comment (Amansingla97) [reply config/database.example.yml]: Done

**Comment by Amansingla97** [comment]
@tachyons kindly see and merge :)
let me know if something needs my attention

**Comment by tachyons** [comment]
Remove sample pg database config file as it is no longer relevant

**Comment by tachyons** [comment]
Also docker

**Comment by Amansingla97** [comment]
@tachyons Done
